DNN Platform/DotNetNuke.Abstractions/Logging/ILogInfo.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+namespace DotNetNuke.Abstractions.Logging
+{
+    using System;
+
+    /// <summary>
+    /// The log info.
+    /// </summary>
+    public interface ILogInfo
+    {
+        /// <summary>
+        /// Gets or sets the log GUID.
+        /// </summary>
+        string LogGUID { get; set; }
+
+        /// <summary>
+        /// Gets or sets the log file ID.
+        /// </summary>
+        string LogFileID { get; set; }
+
+        /// <summary>
+        /// Gets or sets the log type key.
+        /// </summary>
+        string LogTypeKey { get; set; }
+
+        /// <summary>
+        /// Gets or sets the log user id.
+        /// </summary>
+        int LogUserID { get; set; }
+
+        /// <summary>
+        /// Gets or sets the log event id.
+        /// </summary>
+        int LogEventID { get; set; }
```

Comment:
Did we decide with `IPortalAliasInfo` that we would standardize on `Id` over `ID` in these new interfaces?  Note that it _did_ create an issue using the class from VB.NET when both `PortalId` and `PortalID` were exposed, so the `Id` implementation will need to be an explicit interface implementation.

```suggestion
        string LogGuid { get; set; }

        /// <summary>
        /// Gets or sets the log file ID.
        /// </summary>
        string LogFileId { get; set; }

        /// <summary>
        /// Gets or sets the log type key.
        /// </summary>
        string LogTypeKey { get; set; }

        /// <summary>
        /// Gets or sets the log user id.
        /// </summary>
        int LogUserId { get; set; }

        /// <summary>
        /// Gets or sets the log event id.
        /// </summary>
        int LogEventId { get; set; }
```

Reply:
I forgot to update that, thanks for catching this. Can we create a stylecop setting that is specific to the Abstractions project to catch these in the future?

Reply:
Fixed in latest code push, resolving thread